### PR TITLE
Common code style

### PR DESCRIPTION
**Pull request by toxuin**
Hello.

This PR does not add any functionality, just cleans up the code.

This makes code more readable and understandable, also since it's done at once, it should be quite consistent across the codebase now.

This cleans out typos, adds spaces to loops, replaces spaces with tabs at the beginning of the line, distinguishes pointers (`int* myVar`) and pointing to a variable (`*myVar`), etc.

I know things like this are a matter of preference, so I am open to critic and ready to change things around.

This is a bigger PR, so it would be great if more than one person would review it.

### CODE REVIEW

File: cpp_utils/BLEAdvertising.cpp [hunk 1]
```diff
@@ -280,28 +279,28 @@ void BLEAdvertisementData::setAppearance(uint16_t appearance) {
...
-			addData(std::string(cdata, 2) + std::string((char *)&uuid.getNative()->uuid.uuid32,4));
+			addData(std::string(cdata, 2) + std::string((char*) &uuid.getNative()->uuid.uuid32, 4));
 			break;
 		}
 
 		case 128: {
 			// [Len] [0x04] [0] [1] ... [15]
 			cdata[0] = 17;
 			cdata[1] = ESP_BLE_AD_TYPE_128SRV_CMPL;  // 0x07
-			addData(std::string(cdata, 2) + std::string((char *)uuid.getNative()->uuid.uuid128,16));
```
Comment (toxuin) [inline cpp_utils/BLEAdvertising.cpp]: Noticed this. Was this intentional? Am I missing something?
`&uuid.getNative()->uuid.uuid128`
vs
`uuid.getNative()->uuid.uuid128`

Comment (chegewara) [reply cpp_utils/BLEAdvertising.cpp]: https://github.com/espressif/esp-idf/blob/a557e8cc14880717f242918a754cbc37d85e036a/components/bt/bluedroid/api/include/api/esp_bt_defs.h#L87-L92

Comment (toxuin) [reply cpp_utils/BLEAdvertising.cpp]: Is it an error in my code or the original?..

Comment (chegewara) [reply cpp_utils/BLEAdvertising.cpp]: In that union uuid16 and uui32 are type uint_t so you have to use & to get address, uint128 is an array so it is already pointer. By passing variable uuint128 from that union you are passing pointer to first element of array. It was correct in this code.

**Pull request merged**